footer.php

```diff
@@ -7,9 +7,9 @@
 ?>
       <footer>
         <div class="pie-prefooter container-fluid m-0">
-          <div class="row">
+          <div class="row py-2">
```

Comment:
Instead of using padding it's probably better to just add the `text-center` class to the row or the container above it.